inst/staticexports/package.R

```diff
@@ -3,9 +3,20 @@ is_loaded <- function(x) {
 }
 
 is_installed <- function(package, version = NULL) {
-  installed <- nzchar(system.file(package = package))
+  installed <- requireNamespace(package, quietly = TRUE)
```

Comment:
`requireNamespace` will load the package as a side effect, which I think is undesirable. We could do something like this:

```R
  installed <- isNamespaceLoaded(package) || nzchar(system.file(package = package))
```